setup.py

```diff
@@ -6,11 +6,11 @@
     packages=find_packages(),
     include_package_data=True,
     version='3.1.0',
-    description='Orderable model and admin',
+    description='Add manual sort order to Django objects via an abstract base class and admin classes.',
     author='Incuna Ltd',
-    author_email='dev@incuna.com',
+    author_email='admin@incuna.com',
     url='https://github.com/incuna/django-orderable',
-    long_description=open('README.rst').read(),
+    long_description=open('README.md').read(),
```

Comment:
Do we want to drop the `long_description`, or should we continue to use `rst`? Or is there another solution?